tests/models/biattentive_classification_network_test.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=invalid-name,protected-access
+import pytest
+import numpy
+
+
+from allennlp.common.testing import ModelTestCase
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.params import Params
+from allennlp.models import Model
+
+
+class BiattentiveClassificationNetworkMaxoutTest(ModelTestCase):
+    def setUp(self):
+        super(BiattentiveClassificationNetworkMaxoutTest, self).setUp()
+        self.set_up_model('tests/fixtures/biattentive_classification_network/maxout_experiment.json',
+                          'tests/fixtures/data/sst.txt')
+
+    def test_bcn_can_train_save_and_load(self):
+        self.ensure_model_can_train_save_and_load(self.param_file)
+
+    def test_batch_predictions_are_consistent(self):
+        self.ensure_batch_predictions_are_consistent()
+
+    def test_forward_pass_runs_correctly(self):
```

Comment:
This gets checked by the `train_save_and_load` test.  If you like it, it's fine to have, but we don't typically have a separate test for this.